packages/account/src/Styles/account.scss

```diff
@@ -133,8 +133,10 @@ $MIN_HEIGHT_FLOATING: calc(
         }
         &-link {
             display: flex;
-            @include typeface(--small-left-normal-red, none);
 
+            > span {
+                @include typeface(--small-left-normal-red, none);
+            }
```

Comment:
Why do we need this? Can't we somehow cover it in `Text` component?

Reply:
You’re right, I removed this.